packages/api/helper/nominationGsheetToDB.js

```diff
@@ -0,0 +1,68 @@
+const { google } = require('googleapis');
+const db = require('../models');
+const rangePar = 'Sheet1';
+
+module.exports = function gsheetToDB() {
+  const client = new google.auth.JWT(
+    process.env.GOOGLE_SHEETS_CLIENT_EMAIL,
+    null,
+    process.env.GOOGLE_SHEETS_PRIVATE_KEY,
+    ['https://www.googleapis.com/auth/spreadsheets']
+  );
+
+  client.authorize(function (err, tokens) {
+    if (err) {
+      console.log(err);
+      return;
+    } else {
+      console.log('Connected to google sheets API!');
+      getRawNominationData(client);
+    }
+  });
+
+  async function getRawNominationData(cl) {
+    const gsapi = google.sheets({ version: 'v4', auth: cl });
+
+    const opt = {
+      spreadsheetId: process.env.SPREADSHEET_ID_PARAMETER,
+      range: rangePar,
+    };
+
+    let data = await gsapi.spreadsheets.values.get(opt);
+    let nominations = data.data.values;
+
+    nominations.forEach((nomination, index) => {
+      if (index !== 0) {
```

Comment:
One more: Maybe we could just slice the header row off of `nominations` before we start the `forEach` loop instead of testing it each time?